TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/200_Assets/Shaders/OutputDepthNormalsTexture.shader

```diff
@@ -0,0 +1,58 @@
+Shader "Hidden/Test/OutputDepthNormalsTexture"
```

Comment:
This shader is used with the OutputTextureFeature to display the Normals texture.